roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -58,6 +58,9 @@ spec:
               name: "cluster"
           env:
             -
+              name: "DC_NAME"
+              value: "{{deploy_name}}"
```

Comment:
Isn't this available as a label? Per the k8s docs you can pull labels from the downward api.. Which version of k8s were you testing on?